content/blog/2023-11-13-announcing-localstack-30-general-availability/index.md

```diff
@@ -0,0 +1,186 @@
+
+### LocalStack Networking initiative
+
+Over the past few years, LocalStack's configuration options have expanded to accommodate a wide range of use cases, particularly in networking configuration. This includes setting up the listen address for the LocalStack container and configuring domain names for services like SQS or OpenSearch. 
+
+However, this expansion resulted in an excess of configuration variables, some of which were redundant or overlapping in functionality. Additionally, there was inconsistency in how these variables were used across different services — some services relied on one set of variables, while others used different ones, and some didn't support configuration adjustments at all.
+
+To simplify and standardize this process, LocalStack 2.0 introduced two new configuration variables: `GATEWAY_LISTEN` and `LOCALSTACK_HOST`. Building on this, LocalStack 3.0 further standardizes configuration across all LocalStack services. As part of this update, we have eliminated several older configuration variables, including `EDGE_PORT`, `EDGE_PORT_HTTP`, `EDGE_BIND_HOST`, `LOCALSTACK_HOSTNAME`, and `HOSTNAME_EXTERNAL`, streamlining the configuration experience and enhancing overall usability.
+
+View our [Networking migration guide](https://discuss.localstack.cloud/t/networking-migration-guide-for-localstack-3-0/588) for more details!
+
+### Miscellaneous
+
+- We’ve massively **improved write performance for DynamoDB** in LocalStack, with our benchmarks showing a 60% faster [`PutItem`](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_PutItem.html) operation. Using higher batch sizes will profit even more from this change, with [`BatchWriteItem`](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_BatchWriteItem.html) now being 6.2x faster for 10 items and 9.9x faster for 25 items. That means if you’re populating a large DynamoDB table, you can now take advantage of much faster insertion times! For the best performance please set `DYNAMODB_IN_MEMORY=1`, but even without this, you should still notice a significant performance improvement.
+- To limit which services can be started by LocalStack, we have (re-)introduced the concept of **Strict Service Loading**, where only the services defined in the `SERVICES` variable, are going to be started. This also lays the foundation for us to provide standalone images for single services or combinations thereof.
+- We’re also currently in the progress of internally refactoring our CloudFormation resource implementations to be more in parity with AWS and also structurally easier for us and contributors to work with. If you’re interested in contributing here, [please reach out to us](mailto:info@localstack.cloud)! 
+
+If you want to know more check out the [detailed release notes on GitHub]()!
```

Comment:
link missing

Reply:
It would be added once the release is out 👍